client/layout/guided-tours/tours/checklist-register-domain-tour.js

```diff
@@ -0,0 +1,134 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import { noop } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Next,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+export const ChecklistRegisterDomainTour = makeTour(
+	<Tour name="checklistRegisterDomain" version="20171219" path="/domains/add" when={ noop }>
+		<Step
+			name="init"
+			arrow="right-top"
+			placement="beside"
+			style={ {
+				animationDelay: '0.7s',
+			} }
+			target=".register-domain-step__search .search-card"
+		>
+			<p>
+				{ translate(
+					'Search engines like Google and Bing prefer custom domain names and place them higher in search results.'
+				) }
+			</p>
+			<Continue target=".domain-suggestion" step="email-step" click hidden />
+			<Continue target=".domain-search-results" step="email-step" click hidden />
+			<ButtonRow>
+				<Next step="email-step">{ translate( 'Got it, thanks.' ) }</Next>
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Got it, thanks!' )` *ES Score: 14*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/register-domain-guided-tour)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).